scripts/run_device_container.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+#This script should be all any device (other than the control computer)
+#needs to run. The Pyro name server and config server should ideally be
+#initialised beforehand (e.g. on the control computer).
+
+#Download the latest docker image from docker hub
+docker pull huntsmanarray/device_startup
+
+#Run the docker container
+#Note that network=host to map container's ports to the hosts directly
+#This is OK for the pis etc. but probably not for the control computer
+docker run huntsmanarray/device_startup:latest --network host
```

Comment:
```suggestion
docker run huntsmanarray/device_startup:latest --network host --privileged -it
```
Is this working without?  `privileged` will allow you to access the device nodes, which it would seem like we need. `it` allows for `interactive` and `terminal` (or something, that's how I remember it).

Reply:
I think I had it working previously without --privileged and I have read that its best to avoid if possible. So, let's test it without to begin with.